swap_meet/item.py

```diff
@@ -1,2 +1,37 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.condition = condition
+        self.id = id if id is not None else uuid.uuid4().int
+
+
+    def __str__(self):
+        """
+        Converts item to string using str().
+        returns: "An object of type Item with id <id value>."
+        Operator overloading
+        """
+        category = self.get_category()
+        return f"An object of type {category} with id {self.id}."
+
+    
+    def get_category(self):
+        """
+        Return a string holding the name of the class
+        return name_of_class(str)
+        """
+        return "Item"
+
+    def condition_description(self):
+        """
+        Return a human-readable description of the item's condition.
+
+        Returns:
+            str: A message describing the condition.
+        """
+        if self.condition <= 1:
+            return "Yikes. Don't wanna touch that."
+        elif self.condition >= 3:
+            return "Gimme more. Good stuff."
+        else:
+            return "Not bad. I'll buy it for a dollar."
```

Comment:
Though functionally sound, I would suggest made making the following change for readability. 
```suggestion
        if self.condition <= 1:
            return "Yikes. Don't wanna touch that."
        elif self.condition == 2:
            return "Not bad. I'll buy it for a dollar."
        else:
            return "Gimme more. Good stuff."
```
With this change, other developers can more readily see the description for `2` and the rest applied to `3` and up.